app/javascript/apiCalls.js

```diff
@@ -0,0 +1,21 @@
+export function patchTimeSlot(slot, talk, csrfToken) {
+  const talkID = talk === null ? '' : talk.id.toString();
+
+  const data = JSON.stringify({
+    time_slot: {
+      program_session_id: talkID
+    }
+  });
+  
+  if (slot.update_path) {
+    return fetch(slot.update_path, {
+      method: "PATCH",
+      headers: {
+        "Content-Type": "application/json",
+        "X-CSRF-Token": csrfToken
+      },
+      credentials: "same-origin",
+      body: data
+    })
+  }
+}
```

Comment:
That red 'Ghostbuster' and return symbol are 'No new line at end of file' indicators. It's good style/practice to have a new line at the end of files, and fortunately your text editor can automate that so you don't have to think about it. I think you both use VS Code? If so, the first answer in the link below should suffice. You may have to open and save each file that has no new line, which you can see by scrolling through the PR.

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

Reply:
This solution was awesome @gavin-love